src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/EntityList/list.html.twig

```diff
@@ -24,6 +24,12 @@
     </div>
     {% endif %}
 
+    {% for type, messages in app.session.flashbag.all %}
```

Comment:
Maybe add a partial we could also use on the edit entity view?

Reply:
https://www.pivotaltracker.com/story/show/162378323